docs/server-setup.md

```diff
@@ -55,9 +55,9 @@ AzerothCore needs certain files extracted from the World of Warcraft client. You
 
 By default you will compile your core with tools and you will get the following executable files: **mapextractor**, **vmap4extractor**, **vmap4assembler**, **mmaps_generator** (.exe on windows).
 
-Place the files with your World of Warcraft binary (wow.exe on windows) and run them.
+Place the files in the same directory as your World of Warcraft binary (wow.exe on windows) and run them one after the other.
```

Comment:
You can run them at the same time. Except vmap4assambler which needs to be run after the extractor.